src/components/pages/events/eventOverview/eventOverview.tsx

```diff
@@ -132,6 +196,57 @@ const EventOverview: React.FC = () => {
                 />
               </LoadingWrapper>
             )}
+            <div
+              className="paginationControls"
+              style={{ marginBottom: '20px' }}>
```

Comment:
For consistency, you should choose between providing styles in the stylesheet file `eventOverview.scss` or using style inline like here. Don't use both className and style on the same element, in general.